File: common/src/main/java/juuxel/adorn/block/VisibleBlockWithEntity.java
package juuxel.adorn.block;

import net.minecraft.world.level.block.BaseEntityBlock;
import net.minecraft.world.level.block.RenderShape;
import net.minecraft.world.level.block.state.BlockState;

public abstract class VisibleBlockWithEntity extends BaseEntityBlock {
    protected VisibleBlockWithEntity(Properties settings) {
        super(settings);
    }

    @Override
    public RenderShape getRenderShape(BlockState state) {
        return RenderShape.MODEL;
    }
}

File: common/src/main/kotlin/juuxel/adorn/block/VisibleBlockWithEntity.kt
package juuxel.adorn.block

import net.minecraft.block.BlockRenderType
import net.minecraft.block.BlockState
import net.minecraft.block.BlockWithEntity

abstract class VisibleBlockWithEntity(settings: Settings) : BlockWithEntity(settings) {
    override fun getRenderType(state: BlockState) = BlockRenderType.MODEL
}
